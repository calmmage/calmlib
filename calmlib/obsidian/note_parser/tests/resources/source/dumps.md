- 001
    - dump item 1
- 002
    - dump item 2
- 003
    - dump item 3
