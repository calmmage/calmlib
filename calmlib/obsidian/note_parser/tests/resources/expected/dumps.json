{
  "items": [
    {
      "type": "dump",
      "text": "001",
      "children": [
        {
          "type": "list_item",
          "text": "dump item 1"
        }
      ]
    },
    {
      "type": "dump",
      "text": "002",
      "children": [
        {
          "type": "list_item",
          "text": "dump item 2"
        }
      ]
    },
    {
      "type": "dump",
      "text": "003",
      "children": [
        {
          "type": "list_item",
          "text": "dump item 3"
        }
      ]
    }
  ]
}